cartridges/int_adyen/cartridge/js/pages/checkout/adyen-cse.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable */
+'use strict';
+
+var util = require('./util'),
+ajax = require('./ajax');
+
+function pad ( number ) {
+    if ( number < 10 ) {
+        return '0' + number;
+    }
+    return number;
+}
+
+/**
+ * @function
+ * @description Initializes Adyen CSE  Billing events
+ */
+function initializeBillingEvents() {
+
+	$('#billing-submit').on('click', function (e) {
+		var radioVal = $('.payment-method-options').find(':checked').val();
+		if ('CREDIT_CARD' == radioVal){
+			e.preventDefault();
+			
+			var creditCard = $('[data-method="CREDIT_CARD"]'),
+	        		selectedCardID = creditCard.find('input[name$="_selectedCardID"]'),
+				encryptedData = $('#dwfrm_billing_paymentMethods_creditCard_encrypteddata'),
+				key = SitePreferences.ADYEN_CSE_JS_PUBLIC_KEY,
+				encryptedDataValue,
+	        		options = {};
+   
+	        /**
+	         * We need encrypt only CVC if we used already saved CC from dropdown list
+	         */
+	        if ($('#creditCard_number').val().indexOf('*') > -1 && selectedCardID != null && selectedCardID.val() !== '') {
```

Comment:
1. isn't the selectedCardID enough to determine if a card is selected?
2. shall we allow also changes in expiration month/year?